Automattic/wp-calypso
# Sharing: Update example images to retina

#### Changes proposed in this Pull Request

* Make sure our example images for each of the sharing services are large enough to display retina-ized for retina screens.

**Before**

![screencapture-wordpress-marketing-connections-calobeetesting87-wordpress-com-2020-08-10-15_44_29](https://user-images.githubusercontent.com/2124984/89824933-d3adae00-db21-11ea-9909-6f8e054ee249.png)

**After**

![screencapture-calypso-localhost-3000-marketing-connections-calobeetesting87-wordpress-com-2020-08-12-10_45_10](https://user-images.githubusercontent.com/2124984/90029557-00300a00-dc89-11ea-8cbb-6645fa3616f9.png)

#### Testing instructions

* Switch to this PR
* Navigate to `/marketing/connections/[site]`
* Ensure all sharing services are disconnected. Click the drop-down for each service to see the example images.
* Check to make sure images are sharp and clear on retina displays.

Fixes #44317
